docs/platform-engineers/addon/intro.md

```diff
@@ -89,9 +88,17 @@ The README of this addon, it will be displayed in the dashboard for end user who
 
 There're no restrict rules for an [experimental addon](https://github.com/kubevela/catalog/tree/master/experimental/addons), but if the addon want to be [verified](https://github.com/kubevela/catalog/tree/master/addons), the README is the most important thing.
 
-### template.yaml(Optional)
+### template file (Optional)
+
+Through the above introduction, we know that a very important part of an addon is resource description file that define the Kubernetes resource objects that needs to be installed. You can write the `template file` (template.yaml or template.cue) or files below `resources/` dir to define these resources. These resources will be rendered into a KubeVela application, which applied to each cluster by the KubeVela controller. So the template file here can be used to define the basic framework of the application. You can use this file to describe the specific information of the application, for example: labels or annotations of the application, and of course you can directly add components, policies and set workflows in the template file. The type of template file can be YAML (template.yaml) or CUE (template.cue).
```

Comment:
Through the above introduction, we know that a very important part of an addon is its resource description files which define the Kubernetes resource objects to be installed.

Reply:
which will be dispatched to clusters by the KubeVela controller once the addon is enabled.